go/service/chat_boxer.go

```diff
@@ -20,6 +20,22 @@ import (
 	"github.com/keybase/go-codec/codec"
 )
 
+var publicEncKey libkb.NaclDHKeyPair
+var publicCryptKey keybase1.CryptKey
+
+func init() {
+	var zero [libkb.NaclDHKeySecretSize]byte
+	var err error
+	publicEncKey, err = libkb.MakeNaclDHKeyPairFromSecret(zero)
+	if err != nil {
+		panic(err)
+	}
+	publicCryptKey = keybase1.CryptKey{
+		KeyGeneration: 1,
+		Key:           keybase1.Bytes32(*publicEncKey.Private),
+	}
```

Comment:
I guess I just don't understand this computation and was suprised we needed any public key crypto.  Can't we just set:

```go
publicCryptKey = keybase1.CryptKey {
   KeyGeneration  : 1, 
   Key : keybase1.Bytes32(zero[:]),
}
```
?